### DISCUSSION TITLE
Request to review/add asdf-ibmcloud

### DISCUSSION THREAD

**Discussion by triangletodd** [asdf-community/.github#47] [Show and tell]
Request to add [asdf-ibmcloud](https://github.com/triangletodd/asdf-ibmcloud) to this community. 

It is based on [asdf-plugin-template](https://github.com/asdf-vm/asdf-plugin-template).

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
